portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +29,23 @@ function change() {
         counter = 0;
     }
 }
+
+function getMessages() {
+    fetch('/data').then(response => response.json()).then((messages) => {
+        const messageListElement = document.getElementById("messages-container");
+        messageListElement.innerHTML = '';
+        messageListElement.appendChild(
+            createListElement(messages[0]));
+        messageListElement.appendChild(
+            createListElement(messages[1]));
+        messageListElement.appendChild(
+            createListElement(messages[2]));
```

Comment:
I'd suggest using a for loop to loop through `messages` and append to `messageListElement`